pkgs/applications/networking/instant-messengers/pidgin-plugins/pidgin-indicator/default.nix

```
@@ -0,0 +1,33 @@
+{ autoreconfHook
+, fetchFromGitHub
+, glib
+, intltool
+, lib
+, libappindicator-gtk2
+, libtool
+, pidgin
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pidgin-indicator";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "philipl";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-CdA/aUu+CmCRbVBKpJGydicqFQa/rEsLWS3MBKlH2/M=";
+  };
+
+  nativeBuildInputs = [ autoreconfHook ];
+  buildInputs = [ glib intltool libappindicator-gtk2 libtool pidgin ];
+
+  meta = {
+    description = "A AppIndicator/KStatusNotifierItem Plugin for Pidgin";
+    homepage = "https://github.com/philipl/pidgin-indicator";
+    maintainers = with lib.maintainers; [ imalison ];
+    license = lib.licenses.gpl2;
+    platforms = with lib.platforms; linux;
```

```suggestion
    platforms = platforms.linux;
```
Should also run on other platforms as well, no?

---

its specifically for a StatusNotifierItem indicator. I suppose that maybe technically that protocol is supported on non-linux platforms (free bsd hurd maybe), but I think I've seen other packages mostly specify linux for dbus/appindicator based stuff.

Certainly not usuable in e.g. darwin.